charts/incubator/plausible/values.yaml

```diff
@@ -39,72 +50,78 @@ plausible:
   # Logging
   LOG_LEVEL: warn
 
-securityContext:
-  readOnlyRootFilesystem: false
-
-podSecurityContext:
-  runAsUser: 1000
-  runAsGroup: 1000
-
-envFrom:
-  - configMapRef:
-      name: '{{ include "tc.common.names.fullname" . }}-env-config'
-  - secretRef:
-      name: '{{ include "tc.common.names.fullname" . }}-env-secret'
-
-installContainers:
-  plausible-install:
-    image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-    envFrom:
-      - configMapRef:
-          name: '{{ include "tc.common.names.fullname" . }}-env-config'
-      - secretRef:
-          name: '{{ include "tc.common.names.fullname" . }}-env-secret'
-    command: [sh]
-    args:
-      - -c
-      - >-
-        /entrypoint.sh db createdb &&
-        /entrypoint.sh db migrate &&
-        /entrypoint.sh db init-admin
-
-upgradeContainers:
-  plausible-upgrade:
-    image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-    envFrom:
-      - configMapRef:
-          name: '{{ include "tc.common.names.fullname" . }}-env-config'
-      - secretRef:
-          name: '{{ include "tc.common.names.fullname" . }}-env-secret'
-    command: [sh]
-    args:
-      - -c
-      - /entrypoint.sh db migrate
+workload:
+  main:
+    podSpec:
+      initContainers:
+        db-create:
+          type: install
+          enabled: true
+          imageSelector: image
+          securityContext:
+            runAsUser: 1000
+            runAsGroup: 1000
+            readOnlyRootFilesystem: false
+          envFrom:
+            - configMapRef:
+                name: plausible-config
+            - secretRef:
+                name: plausible-secret
+          command: [sh]
```

Comment:
fyi, if its a single item, you can leave it as string.
But if you leave it as list shouldn't it be `["sh"]` ?
(or yaml list)